Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Autolab - autograding docker image
```

Comment:
I should have mentioned this earlier but I didn't catch it:

Normally, when a project ships a Dockerfile in the root of the repository, it signals that the app is "Dockerized," i.e., you can use Docker to actually deploy Tango. Since we're using Docker as a part of Tango (not as a way to deploy it consistently), we should move this file somewhere more appropriate (perhaps to `vmms/`?).